tests/MongoDB.Driver.Core.Tests/ChangeStreamDocumentTests.cs

```diff
@@ -0,0 +1,262 @@
+﻿/* Copyright 2018-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Serializers;
+using MongoDB.Bson.Tests.Serialization;
+using Moq;
+using Xunit;
+
+namespace MongoDB.Driver
+{
+    public class ChangeStreamDocumentTests
```

Comment:
Apparently we didn't have any tests for ChangeStreamDocument before, except to the extent that the ChangeStreamDocumentSerializer tests used to indirectly test ChangeStreamDocument.